### ISSUE TITLE
journal always sets `SCREENSHOT_DIR='SCREENSHOT_DIR'`

### ISSUE THREAD

**Issue by KiNaudiz** [deterenkelt/watchsh#12]
watch.sh 20170105 seems to have a bug where it sets `SCREENSHOT_DIR='SCREENSHOT_DIR` for every keyword, despite giving it a custom directory.
First off, my alias:
```
alias wa='watch.sh \
          -d /mnt/storage/naudiz/ \
          --mplayer-opts "--fs  --save-position-on-quit" \
          --heuristics-level 1 \
          --last-ep \
          --remember-sub-and-audio-delay \
          --screenshot-dir /home/naudiz/Pictures/screenshots/ \
          --subfolders "season %keyword disk disc cd part pt dvd" '
```
Now, I use a now keyword:
```
> wa -c evangelion 
* No appropriate directory for screenshots found.
Type a long, correct name to create one or press <Enter> to skip > evangelion
...
```
The video starts, I press `q` to exit and the resulting entry in `.watch.sh/journal` looks like this:
```
KEYWORD='evangelion'
KEYWORD_FIND_PATTERNS='( -iname *evangelion* )'
FIXED_STRING=f
MODE='episodes'
BASEPATH='/mnt/storage/naudiz/'
FIRST_MATCH='Neon Genesis Evangelion'
SUBFOLDERS='/'
VIDEOFILES='... lots of files ...'
VIDEOFILES_COUNT=32
VIDEO_NUMBER=1
declare -a EP_NUMBERS=([0]="1" [1]="2" [2]="3" [3]="4" [4]="5" [5]="6" [6]="7" [7]="8" [8]="9" [9]="10" [10]="11" [11]="12" [12]="13" [13]="14" [14]="15" [15]="16" [16]="17" [17]="18" [18]="19" [19]="20" [20]="21" [21]="21" [22]="22" [23]="22" [24]="23" [25]="23" [26]="24" [27]="24" [28]="25" [29]="26" [30]="L31" [31]="L32")
INTERRUPTED=t
SCREENSHOT_DIR='SCREENSHOT_DIR'
REMEMBER_SUB_AND_AUDIO_DELAY=t
```

**Comment by deterenkelt** [comment]
I’ve tried to start a watching cycle with an ‘illya’ keyword and when the script suggested to create a directory, I typed ‘fate kaleid prisma illya’. Here’s the journal:

```
# watch.sh journal v20170105

KEYWORD='illya'
KEYWORD_FIND_PATTERNS='( -iname *illya* )'
FIXED_STRING=f
MODE='episodes'
BASEPATH='/home/video/anime/'
FIRST_MATCH='[Tsundere] Fate Kaleid Prisma Illya 2wei [BDRip h264 1920x1080 10bit FLAC] (Specials)'
SUBFOLDERS='/'
VIDEOFILES='[Tsundere] Fate Kaleid Prisma Illya 2wei - SP1 [BDRip h264 1920x1080 10bit FLAC][355D931C].mkv
[Tsundere] Fate Kaleid Prisma Illya 2wei - SP2 [BDRip h264 1920x1080 10bit FLAC][8ED245B9].mkv
[Tsundere] Fate Kaleid Prisma Illya 2wei - SP3 [BDRip h264 1920x1080 10bit FLAC][8DB1FFF2].mkv
[Tsundere] Fate Kaleid Prisma Illya 2wei - SP4 [BDRip h264 1920x1080 10bit FLAC][5DFB937F].mkv
[Tsundere] Fate Kaleid Prisma Illya 2wei - SP5 [BDRip h264 1920x1080 10bit FLAC][D4890105].mkv'
VIDEOFILES_COUNT=5
VIDEO_NUMBER=1
declare -a EP_NUMBERS='([0]="1" [1]="2" [2]="3" [3]="4" [4]="5")'
INTERRUPTED=t
SCREENSHOT_DIR='/home/picts/screens//fate kaleid prisma illya'
REMEMBER_SUB_AND_AUDIO_DELAY=t

```

The line where the SCREENSHOT_DIR is written to the journal is 3060: https://github.com/deterenkelt/watchsh/blob/master/sources/watch.sh#L3060. But I don’t see why the variable expansion doesn’t work. It looks like if 
```
${SCREENSHOT_DIR_ORIG:-$SCREENSHOT_DIR}
```
was
```
${SCREENSHOT_DIR_ORIG:-SCREENSHOT_DIR}
                       ^-----  note the lasck of $
```
I suggest first checking that the `$` is in place and the md5sum for watch.sh matches the original one, it should be `abc2ed76f3da9b48dd8b795190263497`.

**Comment by KiNaudiz** [comment]
The md5sum seems to be correct.
```
> md5sum /usr/bin/watch.sh 
abc2ed76f3da9b48dd8b795190263497  /usr/bin/watch.sh
```
And yes, the `$` is missing:
```
[naudiz@tartaros] in ~
> sed -n 3060p /usr/bin/watch.sh 
		data+="\nSCREENSHOT_DIR='$(escape_for_sed_replacement "${SCREENSHOT_DIR_ORIG:-SCREENSHOT_DIR}")'"
```

The version I use is from your `deter` portage-overlay.

**Comment by deterenkelt** [comment]
I have actually fixed this right away in 833498e73a538ece47be4aa6db63287d8f4e8bc0, but forgot to push changes. Please run `layman -s deter` to update the overlay, there should be a new version available.
